Packages/com.unity.inputsystem/InputSystem/Editor/Internal/EditorHelpers.cs

```diff
@@ -10,6 +12,23 @@ internal static class EditorHelpers
         public static Action<string> SetSystemCopyBufferContents = s => EditorGUIUtility.systemCopyBuffer = s;
         public static Func<string> GetSystemCopyBufferContents = () => EditorGUIUtility.systemCopyBuffer;
 
+        public static void RestartEditorAndRecompileScripts()
+        {
+            // The APIs here are not public. Use reflection to get to them.
+
+            // Delete compilation output.
+            var editorAssembly = AppDomain.CurrentDomain.GetAssemblies().First(x => x.ManifestModule.Name == "UnityEditor.dll");
```

Comment:
It may be cleaner to just do something like:
`var editorAssembly  = typeof(UnityEditor.Editor).Assembly;`

Reply:
We should probably write a test just to make sure that this internal property does not disappear or change. Maybe something that just gets the method, just to make sure its still around.

Reply:
Done